pkg/piecestore/psserver/server_test.go

```diff
@@ -277,6 +283,11 @@ func TestRetrieve(t *testing.T) {
 
 func TestStore(t *testing.T) {
 	ctx := testcontext.New(t)
+	satdb, err := satellitedb.NewInMemory()
```

Comment:
You need to use `satellitedbtest.Run()` for testing against both databases

Reply:
Talking to Alex, he told me that psserver's test cases need to be revisited as the test cases fail randomly. Reason unknown. I tried changing to satellite run, and due to rerunning the same test cases again for postures, I am seeing timeout and stack error. The test cases needs to be refactored. For now I will use in memory as all I need is to generate a PBA, it could have been done locally, but it was decided to use bwagreement's GeneratePBA function, which happens to need access to a db. That is why I had to add newInmemory() function just to satisfy.